polling/frequent/activities.py

```diff
@@ -17,6 +19,17 @@ async def compose_greeting(input: ComposeGreetingInput) -> str:
     while True:
         try:
             result = test_service.get_service_result(input)
```

Comment:
I think we need to change `get_service_result` to `async def` and change everywhere that calls it to use `await` if they don't already

Reply:
the service today is just a synchronous mock of a thing. maybe do this improvement as a follow up enhancement since this is just fixing sample as stated?

Reply:
The problem is that the infrequent equivalent of this calling the same mock does to `await` (I admittedly didn't review this code well enough apparently when it was originally added). I think it's important for _this_ example to `await` the call too because it's important to show that this is an await point from which a cancellation can throw. If you don't want to change the signature on the other side to `async def`, sure we can open a GH issue to do that.

Reply:
understood...fixed!

Reply:
I also made the signature of test_service explicitly async